### PR TITLE
Please promote Tags TCK https://download.eclipse.org/jakartaee/tags/3.0/jakarta-tags-tck-3.0.1.zip

### PR DESCRIPTION
**Pull request by scottmarlow**
Please promote https://download.eclipse.org/jakartaee/tags/3.0/jakarta-tags-tck-3.0.1.zip for TCK challenges:
- https://github.com/jakartaee/tags/issues/255
- https://github.com/jakartaee/tags/issues/256
- https://github.com/jakartaee/platform-tck/issues/1313

### CODE REVIEW

**Comment by scottmarlow** [comment]
@pnicolucci could you please check that this pull request for updating the https://jakarta.ee/specifications/tags/3.0/ page looks correct to you?  
Mostly, you just need to review this link and browse to Specifications and choose Tags (3.0) and look at the updated TCK link.  Thanks!

**Comment by ivargrimstad** [comment]
@scottmarlow Where can I find the file to promote? I just don't want to promote the wrong file

**Comment by arjantijms** [comment]
The file to promote seemingly has not been generated by the JSTL CI: 

https://ci.eclipse.org/jstl

So I wonder where "jakarta-tags-tck-3.0.1.zip" came from then.

**Comment by scottmarlow** [comment]
> @scottmarlow Where can I find the file to promote? I just don't want to promote the wrong file

@ivargrimstad Sorry about that, the TCK zip is https://www.eclipse.org/downloads/download.php?file=/ee4j/jakartaee-tck/jakartaee10/staged/eftl/jakarta-tags-tck-3.0.1.zip

File: tags/3.0/_index.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ existing custom tags with Jakarta Standard Tag Library tags.
 * [Jakarta Standard Tag Library 3.0 Javadoc](./apidocs)
 * [Jakarta Standard Tag Library 3.0 Tagdoc](./tagdocs)
 * [Jakarta Standard Tag Library 3.0 TCK](https://download.eclipse.org/jakartaee/tags/3.0/jakarta-tags-tck-3.0.0.zip)([sig](https://download.eclipse.org/jakartaee/tags/3.0/jakarta-tags-tck-3.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/tags/3.0/jakarta-tags-tck-3.0.0.zip.sha256),[pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))
```
Comment (pnicolucci) [inline tags/3.0/_index.md:38]: @scottmarlow don't we need to update the tck link to 3.0.1?

Comment (pnicolucci) [reply tags/3.0/_index.md:38]: nevermind I see we link it below! Thanks! This looks good!

**Comment by ivargrimstad** [comment]
I am waiting for the Jenkins to come back online after being restarted, but I will promote this and the others as soon as possible.

**Comment by ivargrimstad** [comment]
Done: https://download.eclipse.org/jakartaee/tags/3.0/

**Pull request merged**